src/wat-writer.cc

```diff
@@ -943,8 +943,10 @@ Result WatWriter::ExprVisitorDelegate::OnSimdLaneOpExpr(SimdLaneOpExpr* expr) {
 Result WatWriter::ExprVisitorDelegate::OnSimdShuffleOpExpr(
     SimdShuffleOpExpr* expr) {
   writer_->WritePutsSpace(expr->opcode.GetName());
-  writer_->Writef(" 0x%08x 0x%08x 0x%08x 0x%08x", (expr->val.v[0]), (expr->val.v[1]),
-                  (expr->val.v[2]), (expr->val.v[3]));
+  uint8_t* value_ptr = reinterpret_cast<uint8_t*>(&expr->val);
```

Comment:
Same here, please use `Bitcast`

Reply:
Just replacing `reinterpret_cast` with `Bitcast`?

Reply:
While `uint8_t[16] = Bitcast<v128>` works, the other way around does not appear to be supported.

Reply:
This is probably because `Bitcast` does not handle C arrays properly (it takes a `&&` but it has no overload for `T (&)[N]`.

Reply:
I've changed this to use a `std::array<uint8_t, 16>` instead, which does work with `Bitcast`.